sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample2_RecognizeReceiptsFromUri.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    [LiveOnly]
+    public partial class FormRecognizerSamples
+    {
+        [Test]
+        public async Task RecognizeReceiptsFromUri()
+        {
+            string endpoint = Environment.GetEnvironmentVariable("FORM_RECOGNIZER_ENDPOINT");
+            string apiKey = Environment.GetEnvironmentVariable("FORM_RECOGNIZER_API_KEY");
+
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            string receiptUri = TestEnvironment.JpgReceiptUri;
+
+            #region Snippet:FormRecognizerSample2RecognizeReceiptUri
+            CancellationToken ct1;
```

Comment:
would it be good to add a comment about why the distinction on having 2 cts?

Reply:
Ah, this was a mistake.  Removing - thanks for catching!